README.md

```diff
@@ -6,5 +6,18 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails exists as a framework to combine Ruby with HTML, CSS, and JavaScript to create a web application.
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
+
+Image in submissions folder.
+
+
 1. Describe the flow of a PATCH request as it goes through the app.
+
+User to Server. Server to Controller. Controller to Model. Model to Controller. Assemble at controller. Controller to Server. Server to User.
```

Comment:
This flow is correct, but it could be more specific - after all, the question specifically asks about a PATCH request. What specifically would happen in that case?